syft/execution/placeholder.py

```diff
@@ -27,6 +27,19 @@ def __init__(self, owner=None, id=None, tags: set = None, description: str = Non
         self.expected_shape = tuple(shape) if shape is not None else None
         self.child = None
 
+    def __getattribute__(self, name):
```

Comment:
Remind me what this was for? IIRC, it was proposed as a way to do tracing, but maybe it has other uses too. 🤔

Reply:
This one is for accessing methods that are necessary in the AbstractTensor class (the PlaceHolder is a child of that class).
In case we have ```PlaceHolder > CrypTensor```, we might call methods from CrypTensor that are not necessary for PlaceHolder.
Another way to do this is to hook all methods from CrypTensor (this to be done in the crypten branch), but in that scenario, we might have methods that might not be generic -- like ```get_plain_text``` (this is implemented only in CrypTensor)

Reply:
Medium-term plan for `Placeholder` is to make it a plain wrapper that no longer inherits from `AbstractTensor` and uses roughly this approach to forwarding method calls, so I'm not opposed in general. Just wondering why we need it in `master` if it's currently only used for Crypten, which couldn't be merged into `master` last I knew.

Reply:
I was thinking it is a good idea to sync with master the changes that affect the plans, roles or core functionalities (such that the master will have those changes as fast as possible).

Those changes are some (possible) solutions to some problems we faced when merging some PRs.
I was thinking it might be a good idea to have those changes (that affect the syft core) into the master branch.